Chirp/Networking/NetworkService.swift

```diff
@@ -122,6 +123,22 @@ struct NetworkService {
                                     profilePicture: request.profilePicture)
                     promise(.success(user))
                 }
+                db.collection(Table.users.rawValue)
```

Comment:
add  .whereField(Keys.isContributor.rawValue, isEqualTo: true)

in this case you don't have to fetch all users and iterate over them

Reply:
Well said @stepharski

Reply:
Thank you @stepharski, I'll definitely do it 👍